superset/assets/package.json

```diff
@@ -21,6 +22,12 @@
     "cypress": "cypress",
     "cypress-debug": "cypress open --config watchForFileChanges=true"
   },
+  "jest": {
+    "coverageReporters": [
+      "text",
+      "cobertura"
```

Comment:
I guess, we are not generating `cobertura` report now, if its HTML report then we should remove "cobertura" from here

Reply:
Removed "cobertura".